src/NCronJob/Registry/RuntimeJobRegistry.cs

```diff
@@ -213,7 +213,7 @@ public bool TryRegister(Action<IRuntimeJobBuilder> jobBuilder, [NotNullWhen(fals
     public void RemoveJob<TJob>() where TJob : IJob => RemoveJob(typeof(TJob));
 
     /// <inheritdoc />
-    public void RemoveJob(Type type) => jobWorker.RemoveJobType(type);
+    public void RemoveJob(Type type) => jobWorker.RemoveJobyByType(type);
```

Comment:
JOBYYYY :D

Reply:
I need to buy a new keyboard. Or a new set of fingers. :wink:

Reply:
Fixed.

Thinking a bit about it, I kind of liked `Joby` :wink:

Reply:
We should call our public api like this! `v5` coming!!!!!